payment/templates/payment/email/confirmation-email.html

```diff
@@ -34,7 +35,7 @@
 ${_(u"Here is a brief reminder of the mandatory requirements regarding the proctored exam necessary to obtain the certificate:")}
 <ul>
   <li>${_(u"The proctored exam is organized by the ProctorU provider in a part of the course visible only for students who have selected the certified track")}</li>
-  <li>${_(u"You must verify that your equipment is compatible by <a href='http://www.proctoru.com/testitout/' target='_blank'>clicking here</a>.")}</li>
+  <li>${_(u"You must verify that your equipment is compatible by <a href='{proctoru_url_test}' target='_blank'>{proctoru_url_test}</a>.").format(proctoru_url_test=settings.PROCTORU_URL_TEST)}</li>
```

Comment:
tu as changé la phase affichée qui est maintenant:
"You must verify that your equipment is compatible by https://test-it-out.proctoru.com/"

Reply:
par ailleurs, cette trad ne correspond plus dans le fichier PO en anglais, et la trad francaise non plus